src/board.rs

```diff
@@ -29,11 +29,15 @@ impl Board {
         squares[0] = Self::get_back_rank(Player::White);
         squares[BOARD_SIZE - 1] = Self::get_back_rank(Player::Black);
 
-        Self {
+        let mut board = Self {
             squares,
             moves: HashSet::new(),
             player: Player::White,
-        }
+        };
+
```

Comment:
So I would put code on the line right after `};` Also, we usually return without a blank line above it, but what do you think

Reply:
Yeah I agree